website/docs/reference/integrations/jira-cloud-plugin-installation.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Jira Cloud Integration - Installation
+---
+
+## Prerequisites
+
+### Jira Cloud
+
+- [Jira **Cloud**](https://www.atlassian.com/software/jira)
+
+For Jira Data Center, [see the Jira Server plugin](/integrations/jira_server_plugin_installation)
+
+### Unleash
+
+- Unleash v4 or higher
+
+## Required access levels
+
+### Unleash
+
+You will need an Unleash admin to configure an admin API token. (Unleash Open Source)
+
+For Unleash Enterprise we recommend [Service Tokens](https://docs.getunleash.io/how-to/how-to-create-service-accounts), this is also a requirement for integrating with Change Requests
```

Comment:
I think this should link to the reference documentation instead. We should also link to change requests. The links in this suggestion have not been checked, so they may be wrong. 

```suggestion
For Unleash Enterprise we recommend using a [service account](../service-accounts.md). Service accounts are also required to integrate with [Change Requests](../change-requests.mdx)
```